shell/app/locales/utils.ts

```diff
@@ -47,6 +47,10 @@ export const ENV_MAP = {
   test: i18n.t('test environment'),
   staging: i18n.t('staging environment'),
   prod: i18n.t('prod environment'),
+  DEV: i18n.t('dev environment'),
```

Comment:
I dont know the reason, but I guess lowercase the key where it be used could also resolve this